# How to formulate a design matrix
edgeR

[edgeR user guide](http://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf)

[A guide to creating design matrices for gene expression experiments](https://f1000research.com/articles/9-1444/v1)

DESeq2

[DESeq2::?results](https://rdrr.io/bioc/DESeq2/man/results.html)


[DESeq2 contrast tutorial - markdown](https://github.com/tavareshugo/tutorial_DESeq2_contrasts/blob/main/DESeq2_contrasts.md)

[DESeq2 contrast tutorial - slide](https://docs.google.com/presentation/d/1B9zW1_F-kBqQEu4xqxIJrudYP5DecytYMRR6bY4H6aM/edit#slide=id.p)

[DESeq2 experimental design and interpretation](https://rstudio-pubs-static.s3.amazonaws.com/329027_593046fb6d7a427da6b2c538caf601e1.html)

# Workflow

edgeR

[RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR](https://f1000research.com/articles/5-1408)

[From reads to genes to pathways: differential expression analysis of RNA-Seq experiments using Rsubread and the edgeR quasi-likelihood pipeline](https://f1000research.com/articles/5-1438)

[edgeR user guide](http://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf)


DESeq2

[RNA-Seq workflow: gene-level exploratory analysis and differential expression](https://f1000research.com/articles/4-1070)

[Analyzing RNA-seq data with DESeq2](http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html)


# Tutorial materials

[DIY transcriptomics](https://diytranscriptomics.com/)

[https://hbctraining.github.io/main/](https://hbctraining.github.io/main/)